src/store/modules/banners.js

```diff
@@ -14,35 +14,35 @@ export default {
   state: initialState(),
   getters: {
     all: (state, getters, rootState, rootGetters) => {
-      // Transient alerts created via actions
+      // Transient banners created via actions
 
-      const alerts = state.idList.map(i => state.entries[i])
+      const banners = state.idList.map(i => state.entries[i])
 
-      // Derived state alerts
+      // Derived state banners
 
       const currentGroup = rootGetters['currentGroup/value']
       const isGroupPage = rootGetters['route/isGroupPage']
       if (isGroupPage && currentGroup && currentGroup.awaitingAgreement) {
-        alerts.push({
+        banners.push({
           type: 'awaitingAgreement',
           context: currentGroup.activeAgreement,
           dismissible: true,
         })
       }
       if (isGroupPage && currentGroup && currentGroup.isPlayground) {
-        alerts.push({
+        banners.push({
           type: 'playgroundGroupInfo',
           dismissible: false,
           desktopOnly: true,
         })
       }
 
-      return alerts
+      return banners
     },
   },
   actions: {
-    create ({ commit }, alert) {
-      return commit('create', alert)
+    create ({ commit }, banner) {
```

Comment:
Is this still used anywhere?

Reply:
You mean banners that are created by actions?
No, but maybe in the future...

Reply:
I'm usually in favor of removing unused code, especially when it's untested. It's not lost, if the need arises one could still get the from the file history.

Reply:
Fair enough. Done in 7ddeda3